chain/blockchain.go

```diff
@@ -144,22 +144,32 @@ func (b BlockChains) TonChains() map[uint64]ton.Chain {
 type ChainSelectorsOption func(*chainSelectorsOptions)
 
 type chainSelectorsOptions struct {
-	family    string
-	excluding []uint64
+	// Use map for faster lookups
+	familiesToInclude  map[string]struct{}
+	chainSelsToExclude map[uint64]struct{}
```

Comment:
Perhaps this naming would be a bit more concise

```suggestion
	includedFamilies  map[string]struct{}
	excludedChainSels map[uint64]struct{}
```

Reply:
nice one, naming is so hard for me.